Frontend/package.json

```diff
@@ -7,11 +7,19 @@
   "author": "GraphQLBase",
   "license": "MIT",
   "scripts": {
-    "up": "webpack --mode development",
-    "dev": "node ./dist/main.js",
-    "lint": "eslint src/**",
+    "build": "webpack --mode development",
+    "server": "node ./dist/server.js",
+    "up": "yarn build",
+    "lint": "eslint src/**/*.*",
     "test": "jest",
-    "build:production": "webpack --mode production"
+    "build:production": "webpack --mode production",
+    "dev": "run-p dev:nodemon dev:browsersync dev:watch",
+    "dev:nodemon": "nodemon --watch dist ./dist/server.js",
+    "dev:browsersync": "browser-sync start --proxy localhost:3002 --port 5000 --notify",
+    "dev:watch": "chokidar \"+(src|test)/**/*\" -c \"yarn dev:start\"",
+    "dev:start": "run-s dev:prepare dev:reload",
```

Comment:
`run-s` stands for `run serial`. It makes sure that we don't reload the browser until the build phase is completed.